### ISSUE TITLE
gosrc.org: add "f" (search)

### ISSUE THREAD

**Issue by sqs** [sourcegraph/sourcegraph.com#372]
It pops up a search box for the current package's definitions.

**Comment by shurcooL** [comment]
Do you mean like https://github.com/golang/gddo/pull/128 (aka https://github.com/golang/gddo/pull/242)?

**Comment by sqs** [comment]
Yeah, exactly that. It should just be a matter of using the same JS code.
